zetta_utils/mazepa_layer_processing/common/volumetric_apply_flow.py

```diff
@@ -605,8 +606,8 @@ def build_volumetric_apply_flow(  # pylint: disable=keyword-arg-before-vararg
     intermediaries_dir: Optional[str] = None,
     allow_cache: bool = False,
     clear_cache_on_return: bool = False,
-    *args: P.args,
-    **kwargs: P.kwargs,
+    op_args: Optional[tuple] = None,
```

Comment:
Is there a problem with this being an Iterable instead of Tuple?

Reply:
No - I can change it.

Reply:
Oh, wait - there is an issue indeed. If it's Iterable, it doesn't support `len`.

Reply:
If `len` is necessary, we can try [[Collection](https://docs.python.org/3/library/collections.abc.html)]